### DISCUSSION TITLE
Add "Wallet Finder" Feature

### DISCUSSION THREAD

**Discussion by katomm** [cardano-foundation/cardano-org#51] [Ideas]
In this discussion, I would like to expand on the idea of a "wallet finder". Instead of for example featuring one particular wallet we need a clean and simple UI where people just click what they are looking for, this could be:

- starter wallet (which allows you to buy ada directly for example)
- NFT wallet
- desktop wallet
- mobile wallet
- holder wallet
- developer wallet (testnets)
- expert wallet (I'm thinking of Eternl here for example - but this could also include bringing back Daedalus)
- (...)

Interesting side discussion here: Wallets must be added with particular care and in accordance with various guidelines. Since the last thing we probably want is for a scam wallet to get listed and have a major scandal. For me a minimum requirement to get listet is that a wallet is at least 6 months on [developers.cardano.org/showcase](https://developers.cardano.org/showcase) (which has already its own requirements)

**Comment by RayTracedObserver** [comment]
This is an excellent idea. And given that there logically will be more and more wallets developed over time it’s a good idea to give the user the ability to sort wallets like you suggest, depending on desired usage. Otherwise new users looking for their first wallet will be very lost and overwhelmed by the many possibilities and may end up not downloading a wallet at all.

Seems also like a good thing to limit wallets to those existing on developers.cardano.org/showcase.

**Comment by RayTracedObserver** [comment]
First of all, thank you @katomm for having created the wallet section on the what is ada page.

To improve the current wallet page maybe adding little colorful tags like these would improve readability
<img width="137" alt="Screenshot 2024-11-21 at 00 12 07" src="https://github.com/user-attachments/assets/b39a7367-0fd9-44bc-8718-327915177cad">

Replace the color names with the names of the OSs that the wallets are compatible with : iOS, Android, Mac OS, Windows, Linux and also add the attributes that @katomm mentioned (starter wallet, NFT compatible wallet, expert wallet, developer wallet etc...

Colors are good because they really improve readability compared to the current grey mention under each wallet description which isn't very visible.

Then later on work on a system where one uses these tags as filters and you can filter out results which reorganizes the displayed wallets according to the activated filters.
